src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -69,7 +69,7 @@ def run(self):
         self.offsets.clear()
 
         conf = {
-            "bootstrap.servers": self.bootstrap_servers,
+            **self.broker_config,
```

Comment:
This ain' JavaScript my friend, you need to go old fashioned here:

```py
conf = dict({
    "group.id": self.group_id,
    "session.timeout.ms": 6000,
    "session.timeout.ms": 6000,
    "auto.offset.reset": self.initial_offset_reset,
    "auto.offset.reset": self.initial_offset_reset,
    "enable.auto.commit": "false",
    "enable.auto.commit": "false",
    "enable.auto.offset.store": "false",
    "enable.auto.offset.store": "false",
    "enable.partition.eof": "false",
    "enable.partition.eof": "false",
    "default.topic.config": {"auto.offset.reset": self.initial_offset_reset},
    "default.topic.config": {"auto.offset.reset": self.initial_offset_reset},
}, **self.broker_config)
```

or

```py
conf = {
    "group.id": self.group_id,
    "session.timeout.ms": 6000,
    "session.timeout.ms": 6000,
    "auto.offset.reset": self.initial_offset_reset,
    "auto.offset.reset": self.initial_offset_reset,
    "enable.auto.commit": "false",
    "enable.auto.commit": "false",
    "enable.auto.offset.store": "false",
    "enable.auto.offset.store": "false",
    "enable.partition.eof": "false",
    "enable.partition.eof": "false",
    "default.topic.config": {"auto.offset.reset": self.initial_offset_reset},
    "default.topic.config": {"auto.offset.reset": self.initial_offset_reset},
}
conf.update(self.broker_config)
```

Reply:
Thanks @BYK! This is actually valid syntax in Python 3. I will refactor the PR to support Python 2.7 as well :)

Reply:
@alexef - sorry, I didn't know the `**dict` syntax made it into `dict`s in Py3, that's great and thanks for sharing the info! We are working towards moving on to Python 3.6 but we still need to support Py 2.7 for a while more.